order.go

```diff
@@ -0,0 +1,108 @@
+package apiutil
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+var (
+	// OrderASC defines the ascending order keyword
+	OrderASC = "asc"
+	// OrderDESC defines the descending order keyword
+	OrderDESC = "desc"
+)
+
+// OrderBy represents ordering information
+type OrderBy struct {
+	OrderItems []Order
+}
+
+// Order represents ordering information for a single key
+type Order struct {
+	Key       string
+	Direction string
+}
+
+// NewOrderBy constructs a basic order by struct
+func NewOrderBy() OrderBy {
+	return OrderBy{
+		OrderItems: []Order{},
+	}
+}
+
+// NewOrder constructs a basic order struct
+func NewOrder(key string, orderDirection string) Order {
```

Comment:
https://www.youtube.com/watch?v=FYH8DsU2WCk

Also, group same-type declarations:
```go
func NewOrder(key, direction string) Order {
```

Reply:
Dang, haven't heard this one in quite a while!